pkg/storage/badger/badger.go

```diff
@@ -8,7 +8,7 @@ import (
 	"io"
 	"time"
 
-	"github.com/cespare/xxhash"
+	"github.com/cespare/xxhash/v2"
```

Comment:
Similar to go-modules comment above, let's keep this change for #38. 
In the current state, the project keeps using vendorred dependencies populated with `github.com/golang/dep`. Adding modules' semantic imports now will break, Docker images building process.

Reply:
Same as above.